calliope/config/storage_inter_cluster.yaml

```diff
@@ -0,0 +1,92 @@
+constraints:
+  storage_max:
+    active: False
```

Comment:
should this be "active" or "enabled"/"exists"/...?

This could cause confusion where "active" means something different in the optimisation world (i.e., whether a constraint was ultimately bounding or not).